utils/heap/queue.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package heap
+
+import (
+	stdheap "container/heap"
```

Comment:
Is this alias needed in a future PR? Generally I try to only alias imports when required

Reply:
Was needed previously because it had a collision with the `heap.Interface` I implemented (later deleted). Not needed now, removed.